### PR TITLE
feat(docs): Add docs on operators [fixes LNG-216]

### PR DESCRIPTION
**Pull request by InversionSpaces**

### CODE REVIEW

File: docs/aqua-book/language/values.md [hunk 1]
```diff
@@ -123,9 +122,78 @@ func foo(a: i32, b: i32) -> i32:
...
+  -- anywhere an integer is required
+  <- a + (b - c) + d * (e - 6)
+```
+
+Precedence of operators from highest to lowest:
+- `**`
+- `*`, `/`, `%` (left associative)
+- `+`, `-` (left associative)
+
+## Comparison operators
```
Comment (folex) [inline docs/aqua-book/language/values.md:135]: Aqua also supports `==` for any values, maybe mention that?

Comment (InversionSpaces) [reply docs/aqua-book/language/values.md:135]: For now it supports it only inside `if` and it is mentioned in docs on `if`

Comment (folex) [reply docs/aqua-book/language/values.md:135]: Well, you have section title as `Comparison operators`. `==` is surely one of them.

Comment (InversionSpaces) [reply docs/aqua-book/language/values.md:135]: Added a note

**Review by DieMyst — APPROVED** [review]

**Pull request merged**